.github/workflows/e2e.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
        # should be compatible with them.
        kube-version:
        - "1.21"
-       - "1.25"
+       - "1.27"
```

Comment:
Shall we do it on 2 steps? That the tests for 1.26 did run at least once? Just for completion :)

Reply:
From [this comment](https://github.com/os-observability/tempo-operator/issues/385#issuecomment-1508041834), I supposed is not needed. Also, our thinking is that: if it works with the older and newer versions... we assume it works with the versions between.